diff --git a/node_modules/@cosmjs/launchpad/build/gas.js b/node_modules/@cosmjs/launchpad/build/gas.js
index 95d5547..3d24d32 100644
--- a/node_modules/@cosmjs/launchpad/build/gas.js
+++ b/node_modules/@cosmjs/launchpad/build/gas.js
@@ -9,11 +9,11 @@ class GasPrice {
         this.denom = denom;
     }
     static fromString(gasPrice) {
-        const matchResult = gasPrice.match(/^(?<amount>.+?)(?<denom>[a-z]+)$/);
+        const matchResult = gasPrice.match(/^(.+?)([a-z]+)$/);
         if (!matchResult) {
             throw new Error("Invalid gas price string");
         }
-        const { amount, denom } = matchResult.groups;
+        const [amount, denom] = matchResult;
         if (denom.length < 3 || denom.length > 127) {
             throw new Error("Gas price denomination must be between 3 and 127 characters");
         }
