diff --git a/node_modules/@gorhom/bottom-sheet/src/.DS_Store b/node_modules/@gorhom/bottom-sheet/src/.DS_Store
new file mode 100644
index 0000000..dbd07c1
Binary files /dev/null and b/node_modules/@gorhom/bottom-sheet/src/.DS_Store differ
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/.DS_Store b/node_modules/@gorhom/bottom-sheet/src/components/.DS_Store
new file mode 100644
index 0000000..d0ae04f
Binary files /dev/null and b/node_modules/@gorhom/bottom-sheet/src/components/.DS_Store differ
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index 74442cd..aec667a 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -1609,6 +1609,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
                     key="BottomSheetRootDraggableView"
                     style={contentContainerStyle}
                   >
+                    {/* @ts-ignore */}
                     {typeof Content === 'function' ? <Content /> : Content}
 
                     {footerComponent && (
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetFooter/BottomSheetFooter.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetFooter/BottomSheetFooter.tsx
index fff8d64..df18029 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetFooter/BottomSheetFooter.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetFooter/BottomSheetFooter.tsx
@@ -61,6 +61,7 @@ function BottomSheetFooterComponent({
       onLayout={handleContainerLayout}
       style={containerStyle}
     >
+      {/* @ts-ignore */}
       {typeof children === 'function' ? children() : children}
     </Animated.View>
   ) : null;
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetModal/BottomSheetModal.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetModal/BottomSheetModal.tsx
index 4198cf5..00e2b1a 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetModal/BottomSheetModal.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetModal/BottomSheetModal.tsx
@@ -382,6 +382,7 @@ const BottomSheetModalComponent = forwardRef<
         onChange={handleBottomSheetOnChange}
         onClose={handleBottomSheetOnClose}
         children={
+          // @ts-ignore
           typeof Content === 'function' ? <Content data={data} /> : Content
         }
         $modal={true}
