diff --git a/node_modules/react-native-country-picker-modal/lib/CloseButton.js b/node_modules/react-native-country-picker-modal/lib/CloseButton.js
index 4fb16f6..634a958 100644
--- a/node_modules/react-native-country-picker-modal/lib/CloseButton.js
+++ b/node_modules/react-native-country-picker-modal/lib/CloseButton.js
@@ -54,7 +54,7 @@ CloseButtonIOS.prototype = propTypes;
 CloseButtonAndroid.prototype = propTypes;
 export default Platform.select({
     ios: CloseButtonIOS,
-    android: CloseButtonAndroid,
+    android: CloseButtonIOS,
     web: CloseButtonIOS
 });
 //# sourceMappingURL=CloseButton.js.map
\ No newline at end of file
diff --git a/node_modules/react-native-country-picker-modal/lib/CountryList.js b/node_modules/react-native-country-picker-modal/lib/CountryList.js
index 8816018..27c8f1f 100644
--- a/node_modules/react-native-country-picker-modal/lib/CountryList.js
+++ b/node_modules/react-native-country-picker-modal/lib/CountryList.js
@@ -10,6 +10,7 @@ const styles = StyleSheet.create({
         flex: 1,
         flexDirection: 'row',
         justifyContent: 'space-between',
+        paddingStart: 8,
     },
     letters: {
         marginRight: 10,
@@ -34,6 +35,7 @@ const styles = StyleSheet.create({
     },
     itemCountryName: {
         width: '90%',
+        paddingLeft: 6,
     },
     list: {
         flex: 1,
